yb-voyager/cmd/importData.go

```diff
@@ -214,6 +213,18 @@ func importData(importFileTasks []*ImportFileTask) {
 		utils.ErrExit("Failed to initialize meta db: %s", err)
 	}
 
+	if importDestinationType == TARGET_DB {
+		record, err := GetMigrationStatusRecord()
```

Comment:
We will have to ensure that the `import data` code path continues to work with `import data file` use-case.

Here, it is possible that the MigrationStatusRecord is not inserted in the meta-db (and hence the `record` will be `nil`).

We should insert a `MigrationStatusRecord`, with initial default values, at the time of project creation (where we initialise meta-db as well).

Reply:
Done